dll/shellext/ntobjshex/ntobjfolder.cpp

```diff
@@ -482,7 +482,18 @@ BOOL CNtObjectFolder::IsFolder(const NtPidlEntry * info)
 
 HRESULT CNtObjectFolder::GetInfoFromPidl(LPCITEMIDLIST pcidl, const NtPidlEntry ** pentry)
 {
+    if (!pcidl)
+    {
+        DbgPrint("PCIDL is NULL\n");
+        return E_INVALIDARG;
+    }
+
     NtPidlEntry * entry = (NtPidlEntry*) &(pcidl->mkid);
+    if (!entry)
```

Comment:
This check is completely useless.

Reply:
It's indeed what's fixing the issue faced in the ticket : NULL entry, trying to access it's cb member of the next LoC